### ISSUE TITLE
OpenGL scene viewer

### ISSUE THREAD

**Issue by andrearastelli** [raytracingDevTeam/raytracing#36]
For previewing the scene that has to be rendered, a OpenGL viewer may be useful.

The first implementation can have a simple, unshaded, representation of the scene.

**Comment by andrearastelli** [comment]
Assigned to you, then 😄 

Let's work on this after the release of V3.0 though, this way we can focus 100% only on that, without messing up the raytracer codebase.

**Comment by andrearastelli** [comment]
Must separate this into smaller chunks though, this way we can manage the OpenGL development bit by bit, as we are doing with the raytracer.
Also I may be of some help with something, so after the release of V3.0 we can sit together and plan a course of action for this.
Maybe referring the [real-time rendering book](https://g.co/kgs/ZkpdvV)?

**Comment by Ale32** [comment]
Exactly, it's a _huge_ issue! Let's finish the raytracer first. That book will definitely help, I'm a proud owner of a paper version of it.
